### PR TITLE
Add some more tests and better debugging support

### PR DESCRIPTION
**Pull request by beaulm**
This is what the `.vscode` change allows you to do:
testrunner

You can merge in my changes if you want, or just copy them over manually or whatever floats your boat. Just figured I'd show a different method of making suggestions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
   "description": "Othello game implemented by Paul Jerome",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "coverage": "jest --collectCoverage",
```
Comment (beaulm) [inline package.json:8]: On the command line you can now run `npm run coverage` which will create/update a "coverage" folder in the root of your project. Then open that folder and the "lcov-report" subfolder and open "index.html" in a browser! You can then click on `index.js` to see a line-by-line coverage report!

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
...
+    "watch": "jest --watch"
```
Comment (beaulm) [inline package.json:9]: Once you have all your tests filled in, go to the command line and run `npm run watch` before starting development work and it'll automatically keep running your unit tests as you work! Super hand

File: package.json (hunk 1 above)
Comment (paulthomasjerome) [reply package.json:8]: I saw someone do that in a tutorial and couldn't figure it out, thanks!

File: package.json (hunk 2 above)
Comment (paulthomasjerome) [reply package.json:9]: Will do! Thanks!

**Pull request merged**